pkg/kubelet/cm/libcontainer_manager_mock.go

```diff
@@ -0,0 +1,76 @@
+// +build linux
```

Comment:
IIUC, it is used in the unit test to verify that `Apply` and `Destroy`, etc are called, but not much test coverage beyond that. Not saying it's completely not useful, but could probably be dropped due to limited test coverage.